js/_api.js

```diff
@@ -143,7 +143,7 @@ export function create(name) {
 				callback(data);
 			}
 			switch (data.status) {
-				case 'Complete':
+				case 'Complete' || 'Completed':
```

Comment:
things from the legacy git resque jobs, like this:

```
Resque_Job_Status::STATUS_COMPLETE => "Complete",
```

I didn't wanna muck around with this at this point :P since that would require some deep thinking and testing of current version of the deploy, admin, git fetch etc.